subsys/bluetooth/mesh/shell.c

```diff
@@ -2659,16 +3208,13 @@ SHELL_STATIC_SUBCMD_SET_CREATE(mesh_cmds,
 #if defined(CONFIG_BT_MESH_LOW_POWER)
 	SHELL_CMD_ARG(lpn, NULL, "<value: off, on>", cmd_lpn, 2, 0),
 	SHELL_CMD_ARG(poll, NULL, NULL, cmd_poll, 1, 0),
-#endif
-#if defined(CONFIG_BT_MESH_GATT_PROXY)
-	SHELL_CMD_ARG(ident, NULL, NULL, cmd_ident, 1, 0),
 #endif
 	SHELL_CMD_ARG(dst, NULL, "[destination address]", cmd_dst, 1, 1),
 	SHELL_CMD_ARG(netidx, NULL, "[NetIdx]", cmd_netidx, 1, 1),
 	SHELL_CMD_ARG(appidx, NULL, "[AppIdx]", cmd_appidx, 1, 1),
 
 	/* Commands which access internal APIs, for testing only */
-	SHELL_CMD_ARG(net-send, NULL, "<hex string>", cmd_net_send, 2, 0),
+	SHELL_CMD_ARG(net-send, NULL, "<ttl> <data>", cmd_net_send, 3, 0),
```

Comment:
There's already a `ttl` command that configures the default TTL value, use that instead of changing this command?